server/audit_math/minerva.py

```diff
@@ -37,6 +37,39 @@ def make_arlo_contest(tally, num_winners=1, votes_allowed=1):
     )
 
 
+def fix_third_party_landslides(contest: Contest) -> Contest:
+    """Take an arlo contest and return a contest with all 0-vote candidates removed.
+
+    Use this function to clean contests before running minerva math on them. Athena
+    will throw a ValueError if it finds a margin between candidates of 0 or 1.
+    """
+
+    name = contest.name
+    ballots = contest.ballots
+    num_winners = contest.num_winners
+    votes_allowed = contest.votes_allowed
+    votes = {}
```

Comment:
I'm a little surprised the typechecker doesn't complain about these assignments

Reply:
I'm not sure if I know what you're referring to, is it that I could just pass these values directly to the constructor? I must admit that I wrote this section before I had coffee so my mind was a bit sluggish 😅

Reply:
Nothing significant to my idle musings!